modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenstrainBase.C

```diff
@@ -13,11 +13,10 @@ validParams<ComputeThermalExpansionEigenstrainBase>()
 {
   InputParameters params = validParams<ComputeEigenstrainBase>();
   params.addCoupledVar("temperature", "Coupled temperature");
-  params.addParam<Real>("stress_free_temperature",
-                        "Reference temperature at which there is no "
-                        "thermal expansion for thermal eigenstrain "
-                        "calculation");
-
+  params.addCoupledVar("stress_free_temperature",
+                       "Reference temperature at which there is no "
```

Comment:
I'd drop that change. The metadata provided by MOOSE automatically makes this redundant.